src/pq_flash_index.cpp

```diff
@@ -478,6 +502,41 @@ inline int32_t PQFlashIndex<T, LabelT>::get_filter_number(const LabelT &filter_l
     return idx;
 }
 
+template <typename T, typename LabelT>
```

Comment:
do you have experiment result to show the improvement?

Reply:
yes, will add you to the thread.